README.md

```diff
@@ -1,251 +1,152 @@
 # 🪪 @pagopa/io-react-native-wallet
 
-📲 Provide data structures, helpers, and API to Wallet Instance.
+Library which provides a high level abstraction to interact with the IT-Wallet ecosystem via a predefined flows, a set of utilities and helpers. It is designed to be used in [io-app](https://github.com/pagopa/io-app).
+Follows the [eudi-wallet-it-docs](https://github.com/italia/eudi-wallet-it-docs) specifications, currently aligned with version [0.7.1](https://github.com/italia/eudi-wallet-it-docs/releases/tag/0.7.1).
```

Comment:
Perhaps it might be convenient to use an automatic badge that points to the latest release instead of having to update the readme manually. What do you think? https://github.com/badges/shields/issues/4293#issuecomment-549178270

Reply:
I think that might not be accurate since we are not always aligned to the latest release, what to you think?